doc/plugin_server_upstreamca_awscertificatemanager.md

```diff
@@ -0,0 +1,53 @@
+# Server plugin: UpstreamCA "awscertificatemanager"
+
+The `awscertificatemanager` plugin uses a certificate authority from AWS
+Certificate Manager (ACM) Private Certificate Authority (PCA) to sign
+intermediate signing certificates for SPIRE Server.
+
+The plugin accepts the following configuration options:
+
+| Configuration           | Description                                            |
+| ----------------------- | --------------------------------------------           |
+| region                  | AWS Region to use                                      |
+| cert_autority_arn       | ARN of the "upstream" CA certificate                   |
+| ca_signing_template_arn | ARN of the signing template to use for the server's CA |
+| signing_algorithm       | Signing algorithm to use for the server's CA           |
+| assume_role_arn         | (Optional) ARN of an IAM role to assume                |
+
+See [AWS Certificate Manager Private Certificate Authority](https://aws.amazon.com/certificate-manager/private-certificate-authority/) for more details.
+
+A sample configuration:
+
+```
+UpstreamCA "awscertificatemanager" {
+    plugin_data {
+        region = "us-west-2"
+        cert_authority_arn = "arn:aws:acm-pca:us-west-2:123456789012:certificate-authority/12ac02bc-d425-49f7-ab78-570a44972772"
+        ca_signing_template_arn = "arn:aws:acm-pca:::template/SubordinateCACertificate_PathLen0/V1"
+        signing_algorithm = "SHA256WITHECDSA"
+        assume_role_arn = "arn:aws:iam::123456789012:role/my-role"
+    }
+}
+```
+
+SPIRE server requires the following policy for the IAM identity used.
```

Comment:
Sorry for the noob question, but how are the creds normally picked up? I know that the aws library has something built-in, but a little more elaboration on this might be good to document here (e.g. is an instance profile required, can the creds be loaded from env variable, etc). I imagine we'll want to expand on the available options here in the future.

Reply:
👍 awesome, thank you!

Reply:
Done